### DISCUSSION TITLE
Question 104: Question: Restricted records [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#708] [Q&A] [answered]
Description: I wanted to check why the catalogue records on the objects tab of Event IRN 2663 are restricted?

**Comment by rondlg** [comment, marked as answer]
Response: They are Anthropology records, Action isn’t included in the Security Dept. field.

**Comment by zxmlong** [comment]
你好！邮件已收到！
